CDP4PluginPackager.Tests/AppTestFixture.cs

```diff
@@ -0,0 +1,101 @@
+using NUnit.Framework;
```

Comment:
please add the appropriate header and make sure using statements are inside the namespace as per coding convention